src/transactions/simulation/TxSimUtils.cpp

```diff
@@ -177,6 +216,20 @@ generateScaledLiveEntries(std::vector<LedgerEntry>& entries,
             replaceIssuer(newEntry.data.offer().buying, partition);
             replaceIssuer(newEntry.data.offer().selling, partition);
             break;
+        case CLAIMABLE_BALANCE:
+        {
+            replaceIssuer(newEntry.data.claimableBalance().asset, partition);
+            newEntry.data.claimableBalance().balanceID.v0() =
+                generateScaledClaimableBalanceID(
+                    newEntry.data.claimableBalance().balanceID.v0(), partition);
+
+            auto& claimants = newEntry.data.claimableBalance().claimants;
+            for (auto& claimant : claimants)
+            {
+                mutateScaledAccountID(claimant.v0().destination, partition);
+            }
```

Comment:
There is missing one other detail, which is more directly related to CAP-33 then to CAP-23. We aren't mutating the sponsorship information in the ledger entry. I really wanted to write this comment on line 186 but it won't let me.

While you are fixing this, can you also mutate the `signerSponsoringIDs` in account entry? Maybe make these two changes in a different commit.

Reply:
Fixed